executable/Env.re

```diff
@@ -90,38 +140,19 @@ let run =
     multishell
       ? makeTemporarySymlink() : Lwt.return(Directories.globalCurrentVersion);
 
-  switch (shell) {
-  | Bash
-  | Zsh =>
-    Printf.sprintf("export PATH=%s/bin:$PATH", path) |> Console.log;
-    Printf.sprintf("export %s=%s", Config.FNM_MULTISHELL_PATH.name, path)
-    |> Console.log;
-    Printf.sprintf("export %s=%s", Config.FNM_DIR.name, fnmDir) |> Console.log;
-    Printf.sprintf(
-      "export %s=%s",
-      Config.FNM_NODE_DIST_MIRROR.name,
-      nodeDistMirror,
-    )
-    |> Console.log;
-    Printf.sprintf(
-      "export %s=%s",
-      Config.FNM_LOGLEVEL.name,
-      Fnm.LogLevel.toString(logLevel),
-    )
-    |> Console.log;
-  | Fish =>
-    Printf.sprintf("set -gx PATH %s/bin $PATH;", path) |> Console.log;
-    Printf.sprintf("set -gx %s %s;", Config.FNM_MULTISHELL_PATH.name, path)
-    |> Console.log;
-    Printf.sprintf("set -gx %s %s;", Config.FNM_DIR.name, fnmDir)
-    |> Console.log;
-    Printf.sprintf(
-      "set -gx %s %s",
-      Config.FNM_NODE_DIST_MIRROR.name,
-      nodeDistMirror,
-    )
-    |> Console.log;
-  };
```

Comment:
When the log level was added to `Env.re` it was only added for bash and zsh, so with fnm 1.15.0 you can't change the log level with fish, since the log level isn't printed, and therefore isn't sourced.